.github/workflows/ci.yml

```diff
@@ -266,3 +266,90 @@ jobs:
         with:
           name: fluvio-k8-logs
           path: /tmp/flv_sc.log
+  k8_cluster_upgrade:
+    name: Kubernetes cluster upgrade test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [infinyon-ubuntu-bionic]
+        rust: [stable]
+    env:
+      FLUVIO_CMD: true
+      FLV_SOCKET_WAIT: 600
+      FLV_CLUSTER_MAX_SC_VERSION_LOOP: 120
+      FLV_CLUSTER_MAX_SC_NETWORK_LOOP: 60
+      FLV_TEST_CONSUMER_WAIT: 300000
+    steps:
+      - uses: actions/checkout@v2
+      - run: helm version
+      - name: Install ${{ matrix.rust }}
```

Comment:
I like to put `Install Rust ${{ matrix.rust }}` so that the action printout says "Install Rust stable" rather than "Install stable"